editor/editor_export.cpp

```diff
@@ -1351,18 +1351,21 @@ bool EditorExportPlatformPC::can_export(const Ref<EditorExportPreset> &p_preset,
 	return valid;
 }
 
-String EditorExportPlatformPC::get_binary_extension(const Ref<EditorExportPreset> &p_preset) const {
+List<String> EditorExportPlatformPC::get_binary_extension(const Ref<EditorExportPreset> &p_preset) const {
```

Comment:
Should probably be "get_binary_extension**s**" then.

Reply:
Yeah, makes sense, I'll change that.